### PR TITLE
[codegen] Update API Resources

### PR DESCRIPTION
**Pull request by rattrayalex-stripe**
r? @ob-stripe (feel free to change reviewer)
cc @stripe/api-libraries

- [ ] Sanity check for correctness.
- [ ] Ensure appropriate test coverage (ie; write tests).

### CODE REVIEW

File: stripe/object_classes.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
     api_resources.Coupon.OBJECT_NAME: api_resources.Coupon,
     api_resources.CreditNote.OBJECT_NAME: api_resources.CreditNote,
     api_resources.Customer.OBJECT_NAME: api_resources.Customer,
+    api_resources.CustomerBalanceTransaction.OBJECT_NAME: api_resources.CustomerBalanceTransaction,
```
Comment (rattrayalex-stripe) [inline stripe/object_classes.py]: err, @ob-stripe , do we need this?

**Comment by rattrayalex-stripe** [comment]
This was a test.

File: stripe/object_classes.py (hunk 1 above)
Comment (remi-stripe) [reply stripe/object_classes.py]: Why wouldn't we need that line?

Comment (remi-stripe) [reply stripe/object_classes.py]: oh wow I think it's missing in prod, this is a real bug.

Comment (rattrayalex-stripe) [reply stripe/object_classes.py]: yeah I think that's correct 😬

**Pull request closed without merging**